corehq/apps/users/static/users/js/roles_and_permissions.js

```diff
@@ -1,3 +1,4 @@
+/* globals django */
```

Comment:
You can also use `gettext` instead of `django.gettext`. `gettext` is in the allowed list (`django.gettext` isn't, in the hope of pushing us towards one way of doing things).

Reply:
updated